src/main/java/io/debezium/connector/yugabytedb/connection/OpId.java

```diff
@@ -54,7 +54,7 @@ public String getKeyString() {
             return "null";
         }
 
-        return Base64.getEncoder().encodeToString(key);
+        return Arrays.toString(key);
```

Comment:
why this change?

Reply:
This is to bring the `toString` implementations of `OpId` and `CdcsdkCheckpoint` aligned - the `CdcdkCheckpoint#toString` uses this array like representation but over here if we directly printed the encoded string, it becomes difficult for us to verify keys' equality while reading the logs so that's why this change.

Reply:
Don't remember but maybe we did it because of security reasons